Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM itkdev/php8.3-fpm:latest
+
+USER root
+
+# Add rsync
+RUN apt-get update && apt-get --yes install rsync && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Add node
+RUN apt-get -y install curl gnupg
+RUN curl -sL https://deb.nodesource.com/setup_20.x  | bash -
+RUN apt-get -y install nodejs
```

Comment:
```suggestion
RUN curl -sL https://deb.nodesource.com/setup_20.x | bash -
RUN apt-get --yes install nodejs
```

It may also be worth adding a reference to information on how to install `node`, e.g.

```
# https://docs.npmjs.com/downloading-and-installing-node-js-and-npm#using-a-node-installer-to-install-nodejs-and-npm
```

(or wherever you have your information from).